Lib/optparse.py

```diff
@@ -309,6 +309,7 @@ def format_option(self, option):
         result.append(opts)
         if option.help:
             help_text = self.expand_default(option)
+            import textwrap
```

Comment:
Style: move local imports to the top of the block
```suggestion
            import textwrap
            help_text = self.expand_default(option)
```